test/webpacker_test.rb

```diff
@@ -7,7 +7,7 @@
 
 module TestApp
   class Application < ::Rails::Application
-    config.root = File.dirname(__FILE__)
+    config.root = File.join(File.dirname(__FILE__), "test_app")
```

Comment:
Perhaps we can create a `test_helper.rb` and extract these common paths configuration to that? 

```rb
require "minitest/autorun"
class Minitest::Test
  @@app_path = File.join(File.dirname(__FILE__), "test_app")
  # Other common setup here ....
end
```